src/opt-in.ts

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+import { commands, WebviewView, WebviewViewProvider } from "vscode";
+
+type OptInMessage = {
+  command: "opt-in";
+};
```

Comment:
**suggestion:** find a way to remove this interface and its construction for now - it's not necessary (the `postMessage` can just be called with a string or `null` for instance)